src/main/windows/html_email.ts

```diff
@@ -518,61 +519,65 @@ const createContextMenu = (win: BrowserWindow, webContents: WebContents) => {
     const { editFlags } = props
     const hasText = props.selectionText.trim().length > 0
 
-    const defaultActions: {
-      [key: string]: () => MenuItemConstructorOptions
-    } = {
-      separator: () => ({ type: 'separator' }),
-      copy: () => ({
-        id: 'copy',
-        label: tx('global_menu_edit_copy_desktop'),
-        enabled: editFlags.canCopy && hasText,
-        visible: props.isEditable || hasText,
-        click() {
-          if (webContents) {
-            webContents.copy()
-          } else {
-            electron.clipboard.writeText(props.selectionText)
-          }
-        },
-      }),
-      paste: () => ({
-        id: 'paste',
-        label: tx('global_menu_edit_paste_desktop'),
-        enabled: editFlags.canPaste,
-        visible: props.isEditable,
-        click() {
-          webContents.paste()
-        },
-      }),
-      copyLink: () => ({
-        id: 'copyLink',
-        label: tx('menu_copy_link_to_clipboard'),
-        visible: props.linkURL.length !== 0 && props.mediaType === 'none',
-        click() {
-          electron.clipboard.write({
-            bookmark: props.linkText,
-            text: props.linkURL,
-          })
-        },
-      }),
-      copyImage: () => ({
-        id: 'copyImage',
-        label: tx('menu_copy_image_to_clipboard'),
-        visible: props.mediaType === 'image',
-        click() {
-          webContents.copyImageAt(props.x, props.y)
-        },
-      }),
+    const menuItems = []
+
+    if (props.isEditable || hasText) {
```

Comment:
what does `isEditable` mean here?

Reply:
I honestly didn't dig that deep and kinda missed this one. There was also a Paste command I removed, maybe leftovers from Signal?

Reply:
more like leftovers from the context menu npm package I used to add context menus to deltachat. anyway paste makes no sense, because emails are read only.
thats also why I wonder what isEditable means, because in my mind editing means writing which goes against readonly.
But @meganoahj tested it and it worked for him, so might not matter besides being mysterious/confusing.